inc/3rd-party/wp-cloudflare-super-page-cache.php

```diff
@@ -0,0 +1,16 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin uh?' );
+
+add_action( 'admin_enqueue_scripts', function($hook_suffix) {
+	if (
+		! is_admin()
+		||
+		'media_page_imagify-bulk-optimization' !== $hook_suffix
```

Comment:
I believe that @markonikolic985 told me that we need to do that in all imagify admin pages not just only bulk optimization one, right?